src/main/java/sx/blah/discord/api/internal/DiscordUtils.java

```diff
@@ -120,6 +119,11 @@
 	public static final Pattern CUSTOM_EMOJI_PATTERN = Pattern.compile("<?:[A-Za-z_0-9]+:\\d+>?");
 
 	/**
+ 	 * Pattern for naming Discord's custom emoji.
+ 	 */
+ 	public static final Pattern EMOJI_NAME_PATTERN = Pattern.compile("([^A-Za-z0-9_])");
```

Comment:
Is there no length requirement?

Reply:
The Length requirement is 2-32 characters. I check that [here](https://github.com/Techtony96/Discord4J/blob/dev/src/main/java/sx/blah/discord/handle/impl/obj/Guild.java#L849). I don't know how to check length with regex.